From f5cf4fc82ee99d92eba94e8c36ee378f0cc6b600 Mon Sep 17 00:00:00 2001
From: Tomas Henzl <thenzl@redhat.com>
Date: Thu, 18 Sep 2014 14:14:15 -0400
Subject: [scsi] megaraid_sas: Remove unused variables in megasas_instance

Message-id: <1411049668-28841-16-git-send-email-thenzl@redhat.com>
Patchwork-id: 96167
O-Subject: [RHEL7.1 e-stor PATCH 15/28] megaraid_sas: Remove unused variables in megasas_instance
Bugzilla: 1088523
RH-Acked-by: Ewan Milne <emilne@redhat.com>
RH-Acked-by: Rob Evers <revers@redhat.com>

The following patch for megaraid_sas removes some unused variables from the megasas_instance structure.

Signed-off-by: Adam Radford <aradford@gmail.com>
Reviewed-by: Martin K. Petersen <martin.petersen@oracle.com>
Signed-off-by: Christoph Hellwig <hch@lst.de>
(cherry picked from commit 15365a1ed7397f5e3481a3ceaa9c3aa215f69b77)
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/scsi/megaraid/megaraid_sas.h b/drivers/scsi/megaraid/megaraid_sas.h
index 2e4b808..2e2fcb2 100644
--- a/drivers/scsi/megaraid/megaraid_sas.h
+++ b/drivers/scsi/megaraid/megaraid_sas.h
@@ -1633,8 +1633,6 @@ struct megasas_instance {
  struct timer_list sriov_heartbeat_timer;
  char skip_heartbeat_timer_del;
  u8 requestorId;
- u64 initiator_sas_address;
- u64 ld_sas_address[64];
  char PlasmaFW111;
  char mpio;
  int throttlequeuedepth;
-- 
1.7.1